### DISCUSSION TITLE
Nested Facets

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#15130] [Q&A]
Is it possible to do a nested facet query. e.g

Facet for hours of the day and then within hours a facet by some other property so you would end up with nested results.

We can do multiple queries but it would be nice to do it in a single query. 

thanks in advance.

**Comment by ayende** [comment]
No, that isn't something that you can do.
You can do multiple facets at the same time, though, on different properties.

**Reply by krivulcik** [reply]
To retrieve the results more efficiently, you can send the by-hour queries lazily. That would be 24 queries sent in a single batch, saving some network latency, etc.

See also: https://github.com/ravendb/ravendb/discussions/12996

**Reply by mcrio** [reply]
I'd also consider if it's possible to mitigate the problem at UI level. Like not expanding all hours by default but expanding a particular hour on user interaction at which point you'll pull the additional facets.
